src/app/components/OrderBook.tsx

```diff
@@ -86,26 +90,28 @@ function CurrentPriceRow() {
         2
       );
 
-      spreadString = `${spread} (${spreadPercent}%)`;
+      spreadString = `(${spreadPercent}%)`;
```

Comment:
Maybe removing the parentheses?

```suggestion
      spreadString = `${spreadPercent}%`;
```